docs/sdk/features/initialization.md

```diff
@@ -51,5 +51,5 @@ If a variable is first read from the cache and has a listener for [realtime upda
 
 ### [• Java Local SDK](/sdk/server-side-sdks/java-local#installation)
 
-### [• Java Cloud SDK](/sdk/server-side-sdks/java-cloud#installation)
+### [• Java Cloud SDK](/sdk/server-side-sdks/java-cloud/java-cloud-usage#installation)
```

Comment:
I think I steered you wrong on this - new requirement as per slack conversation, we now want this link to point to the 'Getting Started' page

https://taplytics.slack.com/archives/C032ZGWQPCJ/p1681418554428909